Formula/retroshare.rb

```diff
@@ -0,0 +1,48 @@
+class Retroshare < Formula
+  desc "Friend to friend communication platform"
+  homepage "https://retroshare.cc/"
+  url "https://github.com/RetroShare/RetroShare/releases/download/v0.6.5/RetroShare-v0.6.5-source-with-submodules.tar.gz"
+  sha256 "d974d25cbcceccaf6e23e3b8d683d61e91b08c1853b3be0ecb06281809979cb3"
+  head "https://github.com/RetroShare/RetroShare.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "miniupnpc"
+  depends_on "openssl"
+  depends_on "qt"
+  depends_on "rapidjson"
+  depends_on "sqlcipher"
+  depends_on "xapian"
+
+  def install
+    # Avoid RetroShare build system set wrong SDK version
+    inreplace "retroshare.pri",
```

Comment:
these inreplace statements seem like something that should be addressed upstream and not patched in the formula.

Reply:
I agree but RetroShare 0.6.5 have already been released so there will be no changes until next versions (it means very long time to wait)